readthedocs/templates/projects/project_dashboard_base.html

```diff
@@ -77,6 +77,7 @@ <h3>{% trans "Projects" %}</h3>
                               <span class="build-state build-state-passing">{% trans "passing" %}</span>
                             {% else %}
                               <span class="build-state build-state-failing">{% trans "failing" %}</span>
+                              {# TODO: What whould be show here when "not build.sucess and cancelled"? #}
```

Comment:
I'm getting rid of this concept in the new templates. There is no singular status that makes sense here in any regard. That is, we are showing the status of the project based on the `get_latest_build` -- a failure on some minor version can make the entire project look failing.

Instead, the new templates don't show this status at all, but show individual version statuses instead. Even in that case, I don't feel we show the version as "cancelled" though.